src/content/structured/get-started/custom-theme.mdx

```diff
@@ -3,24 +3,29 @@ path: "/get-started/install-components/custom-theme"
 
 navPriority: 9
 
-date: "2024-02-05"
+date: "2024-12-02"
 
 title: "Custom theme"
 
-subTitle: "How to change the theme colour of some components."
+subTitle: "How to change the colours of some components."
 
 contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/get-started/custom-theme.mdx"
 ---
 
+import { useState } from "react";
+import { IcAlert, IcButton, IcTheme } from "@ukic/react";
+
 ## Customising component colours
 
-Some components can have their theme colour changed by using either of the methods below. When the theme colour is changed, components will automatically update.
+Some components can have their 'brand' colour changed by using either of the methods below. When the 'brand' colour is changed, components will automatically update.
 
 Other styles on affected components will also adapt accordingly, such as the colour of text or icons.
 
+Components that follow the 'brand' colour will not update in the same way as other components when adapting to dark mode. Instead their appearance will only adjust to match the contrast of the 'brand' colour.
```

Comment:
Should we have an ic-theme prop table now it's functionality has expanded, to show what the options are?